react-basic-practice/src/pages/dashboard.tsx

```diff
@@ -35,21 +45,30 @@ const Dashboard: React.FC = () => {
   const handleAction = (type: ActionType, id?: number) => {
     const customer = id && data.find((item) => item.id === id);
 
-    customer &&
+    if (type === "Delete" && customer) {
+      handleOpenDeleteModal(customer);
+    } else if (customer) {
       handleOpenModal(
         type === "Edit"
           ? MODAL_TITLES.EDIT_CUSTOMER
           : MODAL_TITLES.VIEW_CUSTOMER,
         customer
       );
+    }
+  };
+
+  const renderModalContent = () => {
+    if (modalTitle === MODAL_TITLES.DELETE_CUSTOMER) {
+      return <Box textAlign="center"><Text>{MODAL_DESCRIPTION.CONFIRM_DESCRIPTION}</Text></Box>
+    }
+
+    if (modalTitle === MODAL_TITLES.VIEW_CUSTOMER && customerData) {
```

Comment:
use switch/case and don't need to check condition customerData

Reply:
i done